### DISCUSSION TITLE
Substract Row

### DISCUSSION THREAD

**Discussion by muksidin** [nocodb/nocodb#7372] [Features]
Hi ..
I am new in nocodb, is there function to substruct row like in Excel? i have attached example picture.
![subsrow](https://github.com/nocodb/nocodb/assets/5665115/18cc935f-4361-4bbb-be95-56e8b5a06d9d)

Thanks.

**Comment by dstala** [comment]
@muksidin NocoDB formula work on data in one records. Doesn't work across records as in example shown above

**Reply by muksidin** [reply]
Okay, thanks for the answer

**Reply by muksidin** [reply]
@dstala any planing to implement this feature for next release?

**Reply by dstala** [reply]
No. This feature is not in our roadmap. Formulas cross referencing records would become complex
